src/Microsoft.TestPlatform.AdapterUtilities/HierarchyConstants.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.AdapterUtilities
+{
+    /// <summary>
+    /// Contants to help declare Hierarchy test property.
+    /// </summary>
+    public static class HierarchyConstants
```

Comment:
I noticed this type name is slightly inconsistent with the naming of `ManagedNameUtilities.Constants`, should we move the managed names up a level in the hierarchy and make the names consistent (esp. since this is a public API).

Reply:
I was thinking about the same thing. At first `Constants` name is too general; I like to change it. My hesitation is this package is technically released as a preview. What do you think?

Reply:
We can change it while it's still a preview. I think we should go ahead and change it to ManagedNameConstants and move it up a level.